core/src/toga/style/pack.py

```diff
@@ -53,15 +57,23 @@
 # Declaration choices
 ######################################################################
 
+# Define here, since they're not available in Travertino 0.3.0
+START = "start"
+END = "end"
```

Comment:
These definitely feel like keywords we should push upstream, even if we can't use them until Travertino 0.4.0 is out.

Reply:
Agreed. I'll put that in shortly.